include/deal.II/matrix_free/mapping_info.h

```diff
@@ -74,7 +74,8 @@ namespace internal
         const UpdateFlags update_flags_cells,
         const UpdateFlags update_flags_boundary_faces,
         const UpdateFlags update_flags_inner_faces,
-        const UpdateFlags update_flags_faces_by_cells);
+        const UpdateFlags update_flags_faces_by_cells,
+        const bool        piola_transform = false);
```

Comment:
Could we be explicit here?
```suggestion
        const bool        piola_transform);
```